pkg/network/ebpf/c/protocols/tls/native-tls.h

```diff
@@ -171,7 +171,10 @@ int uretprobe__SSL_write(struct pt_regs* ctx) {
 
     char *buffer_ptr = args->buf;
     bpf_map_delete_elem(&ssl_write_args, &pid_tgid);
-    tls_process(ctx, t, buffer_ptr, write_len, LIBSSL);
+    conn_tuple_t copy = *t;
+    normalize_tuple(&copy);
+    flip_tuple(&copy);
```

Comment:
please add a comment in code explaining the logic of performing the flip after normalizing it. this isn't straightly clear behavior. also copy that comment to other places with a similar pattern